src/commcare_cloud/ansible/roles/formplayer/templates/logback-spring.xml.j2

```diff
@@ -0,0 +1,42 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
I'm not super familiar with the date formating syntax used here, but does this include the year-month-day implicitly, or only the time component?

Reply:
Ahh my mistake, fixed. Sample
```
2020-06-22 15:13:32 [main] INFO  application.Application - No active profile set, falling back to default profiles: default

```